frontend/src/components/TourDeFoos/TourDeFoos.tsx

```diff
@@ -16,8 +18,22 @@ const theme = createMuiTheme({
 function TourDeFoos() {
   return (
     <MuiThemeProvider theme={theme}>
+
       <Centered>
+        <VerticalSpace height="1rem"/>
+
+        <StartMenuComponent/>
+        <VerticalSpace height="1rem"/>
+
+        <Button variant="contained" color="primary">
```

Comment:
Dobrze jakby te Buttony mialy równą szczerokość. Bo teraz widze jest tak:
![image](https://user-images.githubusercontent.com/18637116/112544127-f641be00-8db6-11eb-96de-37bba6fe5d95.png)

Reply:
Na pewno tak chcemy? Myślisz, ze będzie lepiej?

Reply:
też byłbym za równą szerokością - lepiej to wygląda :-)

Reply:
ok
poprawię